config.js

```diff
@@ -646,6 +646,12 @@ var config = {
     */
     // dynamicBrandingUrl: '',
 
+    // Tileview toolbar button is shown and enabling tileview works even for a single participant.
+    // allowTileviewOnSingleParticipant: false,
```

Comment:
Please keep this separate in another PR, as it's not related to the background alpha.